### PR TITLE
added biom -> metadata transformers and tests

### LINKED ISSUE #185: ENH: transformer: FeatureTable[*] -> Metadata
related to [this closed issue](https://github.com/qiime2/q2-feature-table/issues/152#issuecomment-337659867)

### PR DESCRIPTION
**Pull request by nbokulich**
fixes #185

### CODE REVIEW

**Comment by thermokarst** [comment]
@ebolyen - How do you want to proceed with this? I am inclined to ask @nbokulich to close for now, then we can add the missing axis functionality to the transformer API (and possibly the Metadata API, too), but I am open to alternatives.

**Comment by thermokarst** [comment]
_nervous laughter_

**Comment by thermokarst** [comment]
Discussed out of band with @gregcaporaso & @ebolyen  --- we will put review of this PR on our radar. Regarding the "missing axis functionality" I mentioned above --- we think we can solve that by adding a new method to q2-feature-table: `transpose`.

File: q2_types/feature_table/_transformer.py [hunk 1]
```diff
@@ -60,6 +60,12 @@ def _table_to_dataframe(table: biom.Table) -> pd.DataFrame:
     return pd.DataFrame(array, index=sample_ids, columns=feature_ids)
 
 
+def _table_to_metadata(table: biom.Table) -> qiime2.Metadata:
+    table = _table_to_dataframe(table)
+    table.index.name = '#SampleID'
```
Comment (thermokarst) [inline q2_types/feature_table/_transformer.py]: Can you make this `id` - that way it is applicable when we start working with transposed tables, too?

**Comment by nbokulich** [comment]
Thanks @thermokarst ! I have made the requested change.

**Comment by thermokarst** [comment]
Waiting on a travis re-run - will merge when complete.

**Comment by thermokarst** [comment]
@nbokulich, can you add a note about this to the changelog?

**Pull request merged**